tests/libcxx/expected_results.txt

```diff
@@ -168,6 +168,9 @@ std/utilities/memory/specialized.algorithms/uninitialized.move/ranges_uninitiali
 std/ranges/range.adaptors/range.lazy.split/range.lazy.split.outer.value/ctor.default.pass.cpp FAIL
 std/ranges/range.adaptors/range.lazy.split/range.lazy.split.outer.value/ctor.iter.pass.cpp FAIL
 
+# libc++ doesn't implement LWG-4061
+std/utilities/format/format.functions/bug_81590.compile.pass.cpp FAIL

```

Comment:
This test is weird. It passes `basic_format_context` by value, which is unexpected.